baselines/comparison_with_other_methods_sc.sh

```diff
@@ -240,6 +272,7 @@ segment_sc "${file_t2}" 't2' 'deepseg' '3d'
 segment_sc "${file_t2}" 't2' 'propseg'
 segment_sc_nnUNet "${file_t2}" '2d'
 segment_sc_nnUNet "${file_t2}" '3d'
```

Comment:
Just a quick note, I had to also change this to `3d_fullres` to keep the same naming convention as nnUNet's. (and also I didn't want to rename folders). As a result, the output predictions have the suffix `3d_fullres` instead of just `3d`. 

Please remember to make this change when running `generate_figures.py` !